src/Umbraco.Web/Editors/Filters/UserGroupEditorAuthorizationHelper.cs

```diff
@@ -77,17 +77,17 @@ public Attempt<string> AuthorizeGroupAccess(IUser currentUser, params string[] g
         /// Authorize that the user is not adding a section to the group that they don't have access to
         /// </summary>
         /// <param name="currentUser"></param>
-        /// <param name="currentAllowedSections"></param>
         /// <param name="proposedAllowedSections"></param>
         /// <returns></returns>
         public Attempt<string> AuthorizeSectionChanges(IUser currentUser,
-            IEnumerable<string> currentAllowedSections,
             IEnumerable<string> proposedAllowedSections)
         {
             if (currentUser.IsAdmin())
                 return Attempt<string>.Succeed();
 
-            var sectionsAdded = currentAllowedSections.Except(proposedAllowedSections).ToArray();
+
+            var currentAllowedSections = currentUser.Groups.SelectMany(x => x.AllowedSections).Distinct();
```

Comment:
This doesn't need to be here since `currentUser.AllowedSections` is exactly this and already exists.
I wish I could remember why `currentAllowedSections` was different. The value for that was only the sections on the current group.

Reply:
`currentAllowedSections` was the value of `userGroupSave.PersistedUserGroup.AllowedSections`

but that was equal to the values posted from the client due to the mapper.

Reply:
So is the big problem that the mapper are changing the sections too early? This should only happen if all checks are successeed just before we save the persisted usergroup again?